File: C_Assignment/Assignment_5/programAQ3.c
///////////////////////////////////////////////////////////
//
//  File name : Assigment5Question3.c
//  Descreption : Used to accept accept number from user and print its numbers line.
//  Author : Kartik Ganesh Jare
//  Date : 17/05/25
//  
///////////////////////////////////////////////////////////

#include<stdio.h>

void Dipaly(int iNo)
{
    int iCnt = 0;

    for(iCnt = -iNo; iCnt <= iNo; iCnt++)
    {
        printf("%d",iCnt);   
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number :\n");
    scanf("%d",&iValue);

    Dipaly(iValue);

    return 0;
}

File: Java_Assignment/Assignment_1/programAQ3.java
///////////////////////////////////////////////////////////
//
//  File name : Assigment1Question3.java
//  Descreption : Used to print 5 to 1 numbers on screen
//  Author : Kartik Ganesh Jare
//  Date : 31/05/25
//  
///////////////////////////////////////////////////////////

class Display
{
    public void Display()
    {
        int iCnt = 0;

        iCnt = 5;
        while(iCnt >= 1)
        {
            System.out.println(iCnt);
            iCnt--;
        }    
    }
}

class programAQ3
{
    public static void main(String A[])
    {
        Display dobj = new Display();
        dobj.Display();
    }
}